lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -1474,6 +1477,7 @@ pub struct ConnectParams {
     pub show_desktop_wallpaper: bool,
     pub ad: bool,
     pub nla: bool,
+    pub uuid: [u32; 4],
```

Comment:
The name `uuid` might be confusing - what UUID?

Should we call this something like `client_identifier` instead?

Reply:
You're right, I've changed it.